packages/checkout-ui-extensions/src/components/Spinner/Spinner.ts

```diff
@@ -6,7 +6,7 @@ export interface SpinnerProps {
    * If not specified, it will use the default size which is
    * about twice as big as the base text.
    */
-  size?: 'small' | 'large';
+  size?: 'base' | 'small' | 'large';
```

Comment:
This type change would be an enhancement, not a breaking change as it is additive in nature.